los_keeper/include/los_keeper/wrapper/wrapper.h

```diff
@@ -49,6 +53,7 @@ class Wrapper {
     std::mutex target_state_list;
     std::mutex point_cloud;
     std::mutex control;
+    std::mutex debug_info;
```

Comment:
Whenever you set debug info, you might have to **lock this mutex.**
For example, if you update debug info after planning, 
you should lock this! (of course, it would be very short moment.)

Reply:
@YunwooLee94 Please keep this in mind :)